Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -117,5 +148,21 @@ public void OnInputClicked(InputEventData eventData)
                 anchorManager.AttachAnchor(gameObject, SavedAnchorFriendlyName);
             }
         }
+
+        private void DetermineParent()
+        {
+            if (ParentGameObjectToPlace == null)
+            {
+                if (gameObject.transform.parent == null)
+                {
+                    Debug.LogError("The selected GameObject has no parent.");
+                    PlaceParentOnTap = false;
+                }
+                else
+                {
+                    ParentGameObjectToPlace = gameObject.transform.parent.gameObject;
```

Comment:
Agreed, what if it's nested into a few children?  We should also handle those cases as well.

I think it might be easier to just assign the parent object in the inspector, but this current implementation overwrites whatever you put in the inspector.